### ISSUE TITLE
Missing ngrok dependency? 

### ISSUE THREAD

**Issue by esafirm** [mathdroid/node-line-messaging-api#2]
Hi, i'm new on this Javascript land, is the missing dependency is intentional? Thanks

```
module.js:472
    throw err;
    ^

Error: Cannot find module 'ngrok'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/esa/Documents/nodejs/tembok/node_modules/node-line-messaging-api/build/webhook.js:21:14)
    at Module._compile (module.js:571:32)
    at Module._extensions..js (module.js:580:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/esa/Documents/nodejs/tembok/node_modules/babel-register/lib/node.js:152:7)

```

**Comment by porys** [comment]
try 'npm install --save ngrok'

**Comment by esafirm** [comment]
@porys i already solve the issue, just want to know is it intentional to exclude it in library's `package.json`

**Comment by mathdroid** [comment]
patched in 0.12.1, @esafirm @porys it was not intentional, sorry!
